### ISSUE TITLE
Maximum Number in array

### ISSUE THREAD

**Issue by krutika-ladani** [sherigar/HacktoberFest-2023#170]
I would like to add a code to find maximum number from an array of numbers in Java.

Please assign me this issue @sherigar 
HACTOBERFEST2023

**Comment by meghamsh253** [comment]
I would like to add a code to find maximum number from an array of numbers in Python in 3 different methods.
I did also add my approach below ,please go through it.

Please assign me this issue @sherigar
HACTOBERFEST2023
[largest.txt](https://github.com/sherigar/HacktoberFest-2023/files/12904643/largest.txt)

**Comment by kritikasharma4** [comment]
I would like to contribute to this issue in python language.
Please can you assign this issue to me
HACTOBERFEST2023

**Comment by Ankurcr7** [comment]
@sherigar I would like to contribute to this issue in python language.

**Comment by meghamsh253** [comment]
I made a PR #180 ,Please look into it.
 HACKTOBERFEST 2023

**Comment by Severus25** [comment]
I would like to contribute to this issue in C++ language.
Please can you assign this issue to me
@sherigar

**Comment by liuHongJie1217** [comment]
I would like to use Python for this task, please assign this problem to me.
@sherigar

**Comment by Sgvkamalakar** [comment]
Certainly, I'd like to request the opportunity to address this challenge using Python. @sherigar

**Comment by 1LAV1** [comment]
Certainly, I'd like to request the opportunity to address this challenge using c++. @sherigar

**Comment by 1LAV1** [comment]
hey @sherigar
please assign this problem to me.

**Comment by Sakshipakhale11** [comment]
Hello,I would like to work on this issue. Can you please assign it to me?

**Comment by RutvikaWagh** [comment]
Hello, Can you please assign this task to me.Thank you.

**Comment by Rajiv-paul** [comment]
@sherigar Kindly accept my PR for Maximum Number in Array using java.
HACTOBERFEST2023

**Comment by RohiniSivapalan** [comment]
Added a new approach
